composer.json

```diff
@@ -24,12 +25,12 @@
         "doctrine/coding-standard": "^7.0",
         "doctrine/orm": "^2.6",
         "jdorn/sql-formatter": "^1.1",
-        "phpstan/phpstan": "^0.11",
-        "phpstan/phpstan-deprecation-rules": "^0.11",
-        "phpstan/phpstan-phpunit": "^0.11",
-        "phpstan/phpstan-strict-rules": "^0.11",
-        "phpstan/phpstan-symfony": "^0.11.6",
-        "phpunit/phpunit": "^8.3",
+        "phpstan/phpstan": "^0.12",
+        "phpstan/phpstan-deprecation-rules": "^0.12",
+        "phpstan/phpstan-phpunit": "^0.12",
+        "phpstan/phpstan-strict-rules": "^0.12",
+        "phpstan/phpstan-symfony": "^0.12",
+        "phpunit/phpunit": "^9.0",
```

Comment:
`9.0` requires `php 7.3`, i guess we will have to put `^8.0|^9.0`

Reply:
Since we put the `composer.lock` under version control, we might as well use `^8.0` here…